--- dune-3.24.1/opam/dune.orig
+++ dune-3.24.1/opam/dune
@@ -26,9 +26,6 @@
  (copy dune-private-libs.opam.template ordering.opam.template))
 
 (rule
- (copy dune-private-libs.opam.template dune-rpc-lwt.opam.template))
-
-(rule
  (copy dune-private-libs.opam.template fiber.opam.template))
 
 (rule
--- dune-3.24.1/dune-project.orig
+++ dune-3.24.1/dune-project
@@ -165,17 +165,6 @@
  (description "Library to connect and control a running dune instance"))
 
 (package
- (name dune-rpc-lwt)
- (dir otherlibs/dune-rpc-lwt)
- (synopsis "Communicate with dune using rpc and Lwt")
- (depends
-  (dune-rpc (= :version))
-  (csexp (>= 1.5.0))
-  (lwt (>= 5.6.0))
-  base-unix)
- (description "Specialization of dune-rpc to Lwt"))
-
-(package
  (name dyn)
  (dir otherlibs/dyn)
  (synopsis "Dynamic type")
